controller_manager:
  ros__parameters:
    # only for fake control
    IDM_traj_controller:
      type: joint_trajectory_controller/JointTrajectoryController

# only for fake control
IDM_traj_controller:
  ros__parameters:
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
    joints:
      - drive_joint
    interface_name: position